env/firebase-build.env

```diff
@@ -0,0 +1,7 @@
+REACT_APP_FIREBASE="true"
+REACT_APP_apiKey=AIzaSyAu8t8lLAUZecEp51TZ_LjnM1-vh1OqBpc
+REACT_APP_authDomain=borton-task-msit.firebaseapp.com
+REACT_APP_projectId=borton-task-msit
+REACT_APP_storageBucket=borton-task-msit.appspot.com
```

Comment:
I can't comment on the empty .env files, but make sure they at least have a comment saying why they are empty

Reply:
Is there a way to comment in a .env?

Reply:
I think a line that starts with `#` works